roles/manifest/README.md

```diff
@@ -0,0 +1,69 @@
+theforeman.foreman.manifest
+===========================
+
+Upload Subscription Manifest to Katello
+
+Requirements
+------------
+
+- `theforeman.foreman` collection
+
+Role Variables
+--------------
+
+- `server_url`: URL of the Foreman server.
+- `username`: Username accessing the Foreman server.
+- `password`: Password of the user accessing the Foreman server.
+- `validate_certs`: Whether or not to verify the TLS certificates of the Foreman server.
+- `organization`: Organization to upload Manifest to.
+- `manifest_path`: Where to find the Manifest on Foreman server. When using manifest_download, it is first downloaded to this location.
+- `manifest_download`: Whether to first download the Manifest from the Red Hat Customer Portal.
+- `manifest_uuid`: UUID of the Manifest to download. Most correspond to a Subscription Allocation defined at https://access.redhat.com/management/subscription_allocations
+- `rhsm_username`: Your username for the Red Hat Customer Portal. Required when manifest_download is True.
+- `rhsm_password`: Your password for the Red Hat Customer Portal. Required when manifest_download is True.
+
+Example Playbooks
+-----------------
+
+Use a Subscription Manifest which is already stored on Foreman server at `/root/manifest.zip`:
+
+```yaml
+- hosts: localhost
+  roles:
+    - role: theforeman.foreman.manifest
+      vars:
+        server_url: https://foreman.example.com
+        username: "admin"
+        password: "changeme"
+        organization: "Default Organization"
+        manifest_path: "/root/manifest.zip"
+        manifest_download: False
+```
+
+Download the Subscription Manifest from the Red Hat Customer Portal before uploading to Foreman server:
+
+```yaml
+- hosts: localhost
+  roles:
+    - role: theforeman.foreman.manifest
+      vars:
+        server_url: https://foreman.example.com
+        username: "admin"
+        password: "changeme"
+        organization: "Default Organization"
+        manifest_path: "/root/manifest.zip"
+        manifest_download: True
+        rhsm_username: "happycustomer"
+        rhsm_password: "$ecur3p4$$w0rd"
+        manifest_uuid: "01234567-89ab-cdef-0123-456789abcdef"
+```
+
+License
+-------
+
+GPLv3+
+
+Author Information
+------------------
+
+The Foreman Project
```

Comment:
I think i agree with @evgeni we could drop License and Author INfo here since they are covered at a collection level.